tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html

```diff
@@ -27,31 +27,31 @@
     {% pageurl page.blog_index as blog_index_url %}
 
     {% for page_author in page.authors.all %}
-        {% include "patterns/molecules/author/author.html" with author=page_author.author item=page tag_link_base=blog_index_url %}
+        {% include "patterns/molecules/author/author.html" with author=page_author.author %}
     {% endfor %}
 
-    {% include_block page.body %}
+    <div>
+        {% for tag in page.related_services.all %}
```

Comment:
I don't think we need the `all` here as you add it in the model.

Reply:
I looked at this again, and I think we actually need it -- you may be referring to a single item in the `page.related_blog_posts` list, where, for each item, we just use `item.related_services`, as in https://github.com/torchbox/torchbox.com/blob/b9fd680cccf7c873650507fa96737b700c64b8cd/tbx/project_styleguide/templates/patterns/molecules/blog-item/blog-item.html#L33